OutOfSchool/OutOfSchool.Redis/CacheService.cs

```diff
@@ -18,6 +18,8 @@ public class CacheService : ICacheService, IDisposable
     private readonly bool isEnabled = false;
 
     private readonly object lockObject = new object();
+    
+    private bool _disposed;
```

Comment:
(Code style) Please be consistent with private fields naming. All other fields don't have underscore prefix, so could do `_disposed`. And, IMO, `isDisposed` is even better name for the flag.